java8/src/main/java/cucumber/runtime/java8/Java8HookDefinition.java

```diff
@@ -1,17 +1,17 @@
-package cucumber.runtime.java;
+package cucumber.runtime.java8;
+
+import static java.util.Arrays.asList;
```

Comment:
I've been using the default that comes with IDEA. What does Eclipse do by default? I'd prefer a minimal amount of configuration if possible.